templates/msmtprc.j2

```diff
@@ -14,7 +14,11 @@ tls_trust_file /etc/ssl/certs/ca-certificates.crt
 # Logging
 {% if msmtp_log is defined %}
 {% if msmtp_log == "syslog" %}
+{% if msmtp_syslog_facility %}
+syslog {{msmtp_syslog_facility}}
+{% else %}
 syslog on
+{% endif %}
```

Comment:
There's no need for the checking here as [`syslog` can accept any of the options](https://marlam.de/msmtp/msmtp.html#Commands-specific-to-sendmail-mode):
> ‘syslog [(on|off|facility)]’

```suggestion
syslog {{ msmtp_syslog_facility }}
```